src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -655,6 +681,12 @@ private void cleanUpBuildings(TickEvent.WorldTickEvent event)
         {
             removedBuildings.forEach(AbstractBuilding::destroy);
         }
+
+        List<BlockPos> removedFields = fields.keySet().stream().filter(loc -> world.getTileEntity(loc) == null).collect(Collectors.toList());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'removedFields' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)